### ISSUE TITLE
Expected interaction between WSI and Vulkan implementation

### ISSUE THREAD

**Issue by ncesario-lunarg** [KhronosGroup/Vulkan-Docs#1634]
@kvark brought up an [issue](https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/3262) on Vulkan-ValidationLayers regarding the expected behavior between a vulkan implementation and a WSI implementation. e.g., what is the expected lifetime of a vulkan resource (in this case, vulkan implementation callbacks it registers with the WSI) held onto by the WSI implementation? This ambiguity has led to [crashes](https://github.com/gfx-rs/wgpu/issues/318) with [xlib](https://github.com/KhronosGroup/Vulkan-LoaderAndValidationLayers/issues/1894) (and perhaps other WSIs) in the past.

**Comment by cubanismo** [comment]
As far as I know, this is always either a driver bug or an instance of this bug in Xlib:

[https://gitlab.freedesktop.org/xorg/lib/libxext/-/issues/3](https://gitlab.freedesktop.org/xorg/lib/libxext/-/issues/3)

There have been driver bugs here in the past, but to the best of my knowledge, the above Xlib issue is the last known instance of this issue in the NVIDIA driver at least.

**Comment by ncesario-lunarg** [comment]
Thanks @cubanismo, should there be any spec language added that "spells out" what an implementation _should_ do?

**Comment by stonesthrow** [comment]
Ping @kvark - note feedback from Cubanismo. We don't have guidance on the right thing - outside out purview.

**Comment by ncesario-lunarg** [comment]
Yep, if there is no change that can/will be made, let's close it. Thanks!
